redpanda/resources/network/data_network.go

```diff
@@ -98,6 +98,29 @@ func datasourceNetworkSchema() schema.Schema {
 			"customer_managed_resources": schema.SingleNestedAttribute{
 				Computed: true,
 				Attributes: map[string]schema.Attribute{
+					"gcp": schema.SingleNestedAttribute{
+						Optional: true,
+						Computed: true,
+						Attributes: map[string]schema.Attribute{
+							"network_name": schema.StringAttribute{
+								Computed:    true,
```

Comment:
I left these without `Required`, but not entirely sure if that's right.

Reply:
Should be fine as is a computed field and it's on a data source.